circle.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: 5.3.0
```

Comment:
Cool 👍 , If Circle CI use NVM for NodeJS version management then changing this to `4` or `6` will get you the latest version from each respective branch (and save you from having to update it too often) once you've decided which branch you'll use for the project 😄